gnome-initial-setup/gnome-initial-setup-copy-worker.c

```diff
@@ -55,7 +55,7 @@ move_file_from_homedir (GFile       *src_base,
 
   g_file_make_directory_with_parents (dest_parent, NULL, NULL);
 
-  if (!g_file_move (src, dest, G_FILE_COPY_NONE,
+  if (!g_file_copy (src, dest, G_FILE_COPY_NONE,
```

Comment:
Because we are relying on `gnome-initial-setup-copy-worker` to copy across the configuration into the demo session now. However, when we delete the home directory of the demo user of the logout, all of our configuration data is lost because g-i-s-c-w is no longer able to move it back into place (the original files are gone). So we need to copy instead.